lib/ruby_lsp/ruby_lsp_rails/code_lens.rb

```diff
@@ -79,13 +79,15 @@ class CodeLens
       sig do
         params(
           client: RunnerClient,
+          global_state: T.nilable(GlobalState),
```

Comment:
Can we not use `T.must(@global_state)` when instantiating `CodeLens` to avoid this being nilable?

Reply:
I also agree that it's better to make it non-nilable if possible.

Reply:
But it is nilable. Since the runner client is initialized in a thread, we may receive a code lens request before it's done initializing and in that case it will break.

And it's not that uncommon. If you restart the LSP with files already opened, there's a pretty significant chance that a code lens request will be run before we're done booting the Rails server.

Reply:
But if that's the case, shouldn't we already received many errors due to the incorrect `T.must` or `NoMethodError`?